src/styles/globals.css

```diff
@@ -0,0 +1,48 @@
+body {
+    margin: 0;
+    padding: 0;
+    color: rgba(255, 255, 255, 0.87);
+    background-color: #000000; 
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
**praise**: Good choice of font-family, ensuring compatibility across different browsers and devices.